src/ExceptionRenderer/RendererAbstract.php

```diff
@@ -268,14 +268,12 @@ protected function makeRelativePath(string $path): string
 
     protected function tryRelativizePathsInString(string $str): string
     {
-        $str = preg_replace_callback('~(?<!\w)(?:[/\\\]|[a-z]:)\w?+[^:"\',;]*?\.php(?!\w)~i', function ($matches) {
+        return preg_replace_callback('~(?<!\w)(?:[/\\\]|[a-z]:)\w?+[^:"\',;]*?\.php(?!\w)~i', function ($matches) {
```

Comment:
would be nice to add comment about what this regex does

Reply:
AFAIK, this is even untested (exact behaviour is unasserted), thus help welcomed. It is a little problematic as the repo can be part of another repo and the tests should still pass.